package.json

```diff
@@ -47,16 +47,16 @@
     "babel-register": "^6.4.3",
     "chai": "^3.0.0",
     "chai-string": "1.2.0",
-    "codecov.io": "0.1.6",
```

Comment:
Good call, looks like this is now deprecated: https://github.com/cainus/codecov.io